packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -36,8 +36,10 @@ export class WaferMap extends FoundationElement {
      */
     public readonly waferMapUpdateTracker = new WaferMapUpdateTracker(this);
 
-    @attr
-    public quadrant: WaferMapQuadrant = WaferMapQuadrant.topLeft;
+    @attr({
```

Comment:
I would try and put this all on one line. There are only a few places in our code that span multiple lines, but I don't think putting it on one line will break the build. There are instances that are longer than what you have here with no disabling of prettier.